src/popup/router/pages/ConfirmationSettings.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="popup">
+    <ListItem v-for="{ name, text, value } in list" :key="name" class="confirmation-row">
+      <CheckBox
+        v-if="typeof value === 'boolean'"
+        :value="value"
+        type="radio"
+        name="confirmationValue"
+        @click.native="changeConfirmation(name, !value)"
+        prevent
+      />
+      <div class="mr-auto text-left">
+        <p class="f-16" data-cy="network-name">{{ text }}</p>
```

Comment:
```suggestion
        <p class="f-16">{{ text }}</p>
```
also, is it possible to combine `div` and `p`?